pvlib/tests/test_atmosphere.py

```diff
@@ -88,6 +88,118 @@ def test_gueymard94_pw():
     assert_allclose(pws, expected, atol=0.01)
 
 
+# Unit tests
+def test_rh_from_tdew(
+    tdew_from_rh_tamb, tdew_from_rh_tdew,
+    tdew_from_rh_tdew_aekr, tdew_from_rh_rh,
+    tdew_from_rh_rh_aekr
+):
+
+    # Calculate relative humidity using pandas series as input
+    rh_series = atmosphere.rh_from_tdew(
+        temperature=tdew_from_rh_tamb,
+        dewpoint=tdew_from_rh_tdew
+    )
+
+    # Calulate relative humidity using pandas series as input
+    # with AEKR coefficients
+    rh_series_aekr = atmosphere.rh_from_tdew(
+        temperature=tdew_from_rh_tamb,
+        dewpoint=tdew_from_rh_tdew,
+        coeff=(6.1094, 17.625, 243.04)
+    )
+
+    # Calculate relative humidity using array as input
+    rh_array = atmosphere.rh_from_tdew(
+        temperature=tdew_from_rh_tamb.to_numpy(),
+        dewpoint=tdew_from_rh_tdew.to_numpy()
+    )
+
+    # Calculate relative humidity using float as input
+    rh_float = atmosphere.rh_from_tdew(
+        temperature=tdew_from_rh_tamb.iloc[0],
+        dewpoint=tdew_from_rh_tdew.iloc[0]
+    )
+
+    # test
+    pd.testing.assert_series_equal(
+        rh_series,
+        tdew_from_rh_rh,
+        check_names=False
+    )
+
+    pd.testing.assert_series_equal(
+        rh_series_aekr,
+        tdew_from_rh_rh_aekr,
+        check_names=False
+    )
+
+    np.testing.assert_allclose(
+        rh_array,
+        tdew_from_rh_rh.to_numpy(),
+        atol=0.001
```

Comment:
Nitpick: why `atol=0.001`?  The fixtures are all high precision, so seems like we ought to test at high precision too.

Reply:
Ah, I have removed the atol statement.  I copied an example i saw earlier in the file.